testsuite/tests/multicluster/test_multicluster_dns.py

```diff
@@ -0,0 +1,33 @@
+"""Tests DNS/TLSPolicy across multiple clusters"""
+
+import dns.resolver
+import pytest
+
+pytestmark = [pytest.mark.multicluster]
```

Comment:
there should also be other marks, no? if it is explicitly set in Makefile to not run multicluster tests with other targets. I see at least additional dns and tls marks here

Reply:
For now, the multicluster tests are completely separated from other tests, due to their very different setup and configuration requirements, so I didnt add more marks